### PR TITLE
Minor grammar fixes

### PR DESCRIPTION
**Pull request by wilsonyhlee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -510,9 +510,9 @@ As of 19.May.2022, 18:00:
 
 -   **Perplexity:** This is based on what the model estimates the probability of new data is. The lower the perplexity, the better.  If the model is 100% correct at predicting the next token it will see, then the perplexity is 1. Mathematically this is calculated using entropy. 
 
--   **High-stakes settings:** Such as those identified as "high-risk AI systems" and "unacceptable risk AI systems" in the European Union's proposed [Artificial Intelligence (AI) Act](https://artificialintelligenceact.eu/annexes/).
+-   **High-stakes Settings:** Such as those identified as "high-risk AI systems" and "unacceptable risk AI systems" in the European Union's proposed [Artificial Intelligence (AI) Act](https://artificialintelligenceact.eu/annexes/).
```
Comment (meg-huggingface) [inline README.md:513]: This is a regular phrase, in the glossary -- remove the caps here.

File: README.md [hunk 2]
```diff
@@ -510,9 +510,9 @@ As of 19.May.2022, 18:00:
...
 
--   **Critical decisions**: Such as those defined in [the United States' proposed Algorithmic Accountability Act](https://www.congress.gov/117/bills/s3572/BILLS-117s3572is.pdf).
+-   **Critical Decisions**: Such as those defined in [the United States' proposed Algorithmic Accountability Act](https://www.congress.gov/117/bills/s3572/BILLS-117s3572is.pdf).
```
Comment (meg-huggingface) [inline README.md:515]: This is a regular phrase, in the glossary -- remove the caps here.

File: README.md [hunk 3]
```diff
@@ -510,9 +510,9 @@ As of 19.May.2022, 18:00:
...
```
Comment (meg-huggingface) [inline README.md:516]: I realize "Rights" should be uncapitalized as "rights" as well!

**Review by meg-huggingface — CHANGES_REQUESTED** [review]
Some of these shouldn't be capitalized actually.
Thanks for noticing the others!

**Comment by wilsonyhlee** [comment]
@meg-huggingface Thanks for the feedback. Made all the changes!

**Review by meg-huggingface — APPROVED** [review]
LGTM

**Pull request merged**